spec.html

```diff
@@ -33150,7 +33150,7 @@ <h1>%TypedArray%.prototype.slice ( _start_, _end_ )</h1>
             1. Repeat, while _k_ &lt; _final_
               1. Let _Pk_ be ! ToString(_k_).
               1. Let _kValue_ be ? Get(_O_, _Pk_).
-              1. Perform ! Set(_A_, ! ToString(_n_), _kValue_).
+              1. Perform ! Set(_A_, ! ToString(_n_), _kValue_, *true*).
```

Comment:
811a66e8d4072d230a1b2df5e2440e1101ffb1fb: if the `!` is correct, then "true" would be too - and it seems impossible for this to throw.